docs/form-control.rst

```diff
@@ -94,7 +94,7 @@ properly:
 Hints
 -----
 
-.. php:attr: hint
+.. php:attr:: hint
```

Comment:
example how rendered: https://agile-ui.readthedocs.io/en/stable/form-control.html#hints

(not actually rendered)